taskvine/src/manager/taskvine.h

```diff
@@ -49,8 +49,8 @@ typedef enum {
 } vine_mount_flags_t;
 
 typedef enum {
-	VINE_PEER_SHARE = 1, /**< Schedule this file to be shared between peers where available. (default) **/
-	VINE_PEER_NOSHARE = 2  /**< Do not schedule this file to be shared between peers. **/
+	VINE_PEER_SHARE = 0, /**< Schedule this file to be shared between peers where available. (default) */
```

Comment:
This will not work with comparing flags with "|="

Reply:
My motivation was that for the examples 0 was used, but it is not part of the enumeration. We want 0 to mean something, I think, and it is natural to be the default. If we use 1,2,4, etc. we can do something similar to |=. Would something like the flags in dttools/src/debug.h work for what you are thinking?

E.g., we can change:

`if((m->file->flags |= VINE_PEER_SHARE))`  to   `if(!(m->file->flags  & VINE_PEER_NOSHARE))`. That & checking set bits is also more in line with the rest of the code.

Reply:
Or we can make VINE_PEER_NOSHARE=0 the default.  What do you think?

Reply:
Since we are combining cache flags and sharing flags there will be occasions where that would attempt something like `10000000 & 10100000'